tasks/dogstatsd.py

```diff
@@ -27,7 +27,7 @@
 # constants
 DOGSTATSD_BIN_PATH = os.path.join(".", "bin", "dogstatsd")
 STATIC_BIN_PATH = os.path.join(".", "bin", "static")
-MAX_BINARY_SIZE = 32 * 1024
+MAX_BINARY_SIZE = 35 * 1024
```

Comment:
Is this change related to the PR?

Reply:
Yes, as the size of `dogstatsd` has increased.